tsconfig.json

```diff
@@ -1,112 +1,108 @@
 {
```

Comment:
not sure why this file is using indentation of 4 spaces, though not a block issue!
I will add a `.editorconfig` so that the code style would be aligned among the team.